docs/Training-ML-Agents.md

```diff
@@ -206,7 +206,9 @@ behaviors:
     normalize: false
     num_layers: 2
     time_horizon: 64
+    summary_freq: 10000
```

Comment:
@ervteng Manually applied these from https://github.com/Unity-Technologies/ml-agents/pull/3989/files - can you sanity check me?

Reply:
👍